app/models/miq_task.rb

```diff
@@ -40,7 +40,7 @@ class MiqTask < ApplicationRecord
   def self.update_status_for_timed_out_active_tasks
     MiqTask.active.timed_out.no_associated_job.find_each do |task|
       task.update_status(STATE_FINISHED, STATUS_ERROR,
-                         "Task [#{task.id}] timed out - not active for more than #{::Settings.task.active_task_timeout}")
+                         "Task [#{task.id}] timed out - not active for more than #{::Settings.task.active_task_timeout.tr(".", " ")}")
```

Comment:
Maybe do it this way because, although it doesn't read as nicely, it's safer - 

`"#{::Settings.task.active_task_timeout.to_i_with_method} seconds"`